lib/app_web/views/comment_view.ex

```diff
@@ -14,4 +14,10 @@ defmodule AppWeb.CommentView do
     |> List.first
   end
 
+  def display_markdown(text) do
+    text
+    |> Earmark.as_html!
+    |> raw
```

Comment:
does this check for "_malicious_" HTML or JS tags?
(_or are we relying on GH to "strip" them out?_)